packages/summer-earn-protocol-subgraph/src/mappings/harborCommand.ts

```diff
@@ -197,7 +197,7 @@ function processHourlyVaultUpdate(
 }
 
 export function handleInterval(block: ethereum.Block): void {
-  // ENABLE ONLY for separate subgraph deployment
+  // ENABLE ONLY for separate subgoldsky subgraph deployment
```

Comment:
I know it's maybe not for now, but instead of code uncommenting we should use a parameter to indicate that we want separate deployments